bin/joinL3

```diff
@@ -63,11 +63,11 @@ if __name__ == '__main__':
             # Re-calculate corrected precipitation
             if hasattr(all_ds, 'precip_u_cor'):
                 if ~all_ds['precip_u_cor'].isnull().all():
-                    all_ds['precip_u_cor'],  _ = _correctPrecip(all_ds['precip_u'], 
+                    all_ds['precip_u_cor'],  _ = correctPrecip(all_ds['precip_u'], 
                                                                 all_ds['wspd_u'])
             if hasattr(all_ds, 'precip_l_cor'):
                 if ~all_ds['precip_l_cor'].isnull().all():
-                    all_ds['precip_l_cor'],  _ = _correctPrecip(all_ds['precip_l'], 
+                    all_ds['precip_l_cor'],  _ = correctPrecip(all_ds['precip_l'], 
```

Comment:
> My understanding is that _ preceeding a function name indicates an "internal" function that might just be referenced by another function (e.g. from M import * does not import objects whose name starts with an underscore). This comment applies to all other edits like this in the PR. Maybe this was your intention?

I un-underscored a lot of the functions in `L0toL1`, `L1toL2` and `L2toL3` because they weren't showing up in the readthedocs. But I didn't know about the import * ignoring underscored functions. Good to know.

> Also, just a side comment, but my understanding of general conventions for "CamelCase" is to only use camel case (or "mixed case") for naming classes, not functions. So, this would be ClassName and function_name or module_name. Not worth changing since I believe currently pypromice uses mixed case for function/module names, just thought I would mention it.

Oh, I didn't know that! I think I just use mixed case for everything, but potentially @mankoff used the proper conventions. If this is general, standard conventions though then I would like this changed in the future.